setup.py

```diff
@@ -1,20 +1,30 @@
 from flask import Flask
-from controllers.auth import controller_endpoints
+import os
+from controllers.auth import controller
 from flask_sqlalchemy import SQLAlchemy
 from flask_migrate import Migrate
+from sqlalchemy import VARCHAR, DateTime, text
+from flask_cors import CORS
 
 
 app = Flask(__name__)
-app.config['SQLALCHEMY_DATABASE_URI'] = 'postgresql+psycopg2://jonas:password@localhost:5432/jonas'
+CORS(app)
+app.config['SQLALCHEMY_DATABASE_URI'] = 'postgresql+psycopg2://'+os.getenv('POSTGRES_USER')+':'+os.getenv('POSTGRES_PASSWORD')+'@db:5432/jonas'
 db = SQLAlchemy(app)
 migrate = Migrate(app, db, directory='config/db/postgres/migrations')
 
-class User(db.Model):
+class Users(db.Model):
     id = db.Column(db.Integer, primary_key=True)
-    name = db.Column(db.String(128))
+    username = db.Column(VARCHAR(16), nullable=False)
+    password = db.Column(VARCHAR(64), nullable=False)
+    salt = db.Column(VARCHAR(64), nullable=False)
+    created_at = db.Column(DateTime, nullable=False, server_default=text('NOW()'))
+    updated_at = db.Column(DateTime, nullable=False, onupdate=text('NOW()'))
+    deleted_at = db.Column(DateTime, nullable=True)
+    test = db.Column(VARCHAR(10), nullable=True)
```

Comment:
what's this column meant for?

Reply:
Only test porpuose, but i nedd to delete!